### DISCUSSION TITLE
New lines in tooltip

### DISCUSSION THREAD

**Discussion by mark1502** [saadeghi/daisyui#2687] [Q&A] [answered]
I started integrating DaisyUI tooltips with FullCalendar, but I don't have any ability to format the tooltip.

It seems that the new line code "\n" is ignored, as are hex codes "\x10" and "\x13".  Is there a way to get the new line character recognized in a tooltip?  Also are there any other options available to format a tooltip, such as bg color, text color, etc?

Thanks

**Comment by saadeghi** [comment, marked as answer]
similar discussion: #2655
